src/screens/LoginScreen.js

```diff
@@ -99,44 +117,48 @@ function redirectToSignupScreen(navigation) {
   navigation.replace("Signup");
 }
 
+function redirectToForgotPasswordScreen(navigation) {
+  navigation.replace("ForgotPassword");
```

Comment:
It might be good to put the redirectToForgotPasswordScreen function in another file since it is a reuseable function (used in both LoginScreen and SignupScreen)